### PR TITLE
update FAQ to mention old documentation location

### PR DESCRIPTION
**Pull request by jonniebigodes**
With this pull request the FAQ (workflows/faq.md) page is updated to include versioned Storybook documentation.
I've included the documentation for 5.3 and 5.0 only. 

Addresses and closes issue #12541 

What was done:
- Created a table with the location of the old documentation based on the current documentation structure we have currently setup.
- Some of the links had to be converted into `<a/>` tags to get the proper location. 

Feel free to provide feedback

### CODE REVIEW

File: docs/workflows/faq.md [hunk 1]
```diff
@@ -101,3 +101,63 @@ Starting with Storybook version 6.0 we've introduced some great features aimed a
...
+|                  | CLI options                                | [See current documentation](../api/cli-options.md)                                 | [See versioned documentation](https://github.com/storybookjs/storybook/tree/release/5.3/docs/src/pages/configurations/cli-options)                                                                                                                                                           | [See versioned documentation](https://github.com/storybookjs/storybook/tree/release/5.0/docs/src/pages/configurations/cli-options)                                                                                                                                                           |
+
+
+<div class="aside">
+With the release of version 5.3 we've updated how you can write your stories to a more compact and easier way. It doesn't mean that the <code>storiesOF</code> format has been removed. For the time being we're still supporting it and we have documentation for it. But be advised that this is bound to change in the future.
```
Comment (shilman) [inline docs/workflows/faq.md:162]:
```suggestion
With the release of version 5.3 we've updated how you can write your stories to a more compact and easier way. It doesn't mean that the <code>storiesOf</code> format has been removed. For the time being we're still supporting it and we have documentation for it. But be advised that this is bound to change in the future.
```

**Review by shilman — APPROVED** [review]
Wow, nice one @jonniebigodes ❤️

**Comment by jonniebigodes** [comment]
@shilman no problem. Just glad that i was able to give some more context for users that are still transitioning from a previous version. For now they have a point of reference.

**Comment by jonniebigodes** [comment]
@shilman i've accepted the proposed change. Going to merge it.

**Pull request merged**